### DISCUSSION TITLE
Towny shows: Economy hasn't been turned on

### DISCUSSION THREAD

**Discussion by SecretZudan** [TownyAdvanced/Towny#5658] [Q&A] [answered]
when doing economy related stuff this message comes up and idk how to solve it :/

**Comment by SecretZudan** [comment]
by message i mean: economy hasn't been turned on

**Comment by LlmDl** [comment, marked as answer]
Put Vault and an Economy plugin on to your server.

**Reply by SecretZudan** [reply]
i did that but i have new problem, i don't want money to be digital but idk how to change that. i want to set it to emeralds instead. i want simmilar system as earthmc about economy :)

**Reply by SecretZudan** [reply]
and thanks for helping

**Reply by LlmDl** [reply]
Physical economy plugins that I know of include: TheNewEconomy and Gringotts.

**Reply by SecretZudan** [reply]
i installed new economy but when i check prices in towny it is still digital :/

**Reply by SecretZudan** [reply]
and where in config can i enable it.

**Reply by LlmDl** [reply]
I think that's a question for TheNewEconomy. Read their documentation.

**Reply by SecretZudan** [reply]
and also sorry if there are too many questions but how can i change prices of new land, upkeep and reclaim?

**Reply by LlmDl** [reply]
Towny's Economy section in the config.yml. Consider reading the How Towny Works wiki page. It contains a great many facts.

**Reply by SecretZudan** [reply]
on everything it says: !!org.bukkit.configuration.MemorySection {}
idk what is this and afraid to touch it

**Reply by LlmDl** [reply]
You're using a version of Towny not made for MC 1.18.1

Download the latest pre-release linked on the Towny readme.

**Reply by SecretZudan** [reply]
where is towny readme?

**Reply by LlmDl** [reply]
https://github.com/TownyAdvanced/Towny

**Reply by SecretZudan** [reply]
isn't 0.97.5.0 for 1.18.1?

**Reply by SecretZudan** [reply]
just want to know :)

**Reply by SecretZudan** [reply]
and i also use townychat 0.91

**Reply by SecretZudan** [reply]
what townychat does?

**Reply by LlmDl** [reply]
It says that it does not. Please re-read the documentation that is written so that I don't have to re-answer the same questions over and over.
